Tests/Factory/DiactorosFactoryTest.php

```diff
@@ -161,4 +161,38 @@ public function testCreateResponseFromBinaryFile()
 
         $this->assertEquals('Binary', $psrResponse->getBody()->__toString());
     }
+
+    public function testUploadErrNoFile()
+    {
+        $file = new UploadedFile(null, null, null, 0, UPLOAD_ERR_NO_FILE, true);
```

Comment:
the $path parameter must be a string according to phpdoc, so passing null is invalid